modules/ossm-auto-route-enable.adoc

```diff
@@ -7,6 +7,14 @@
 
 By default, the `ServiceMeshControlPlane` resource automatically synchronizes the Gateway resources with OpenShift routes. Disabling the automatic route creation allows you more flexibility to control routes if you have a special case or prefer to control routes manually.
 
+== Skipping automatic route creation for specific cases
+
+If you want to skip the automatic management of OpenShift routes for a specific Istio Gateway, you must add the annotation `maistra.io/manageRoute: false` to the Gateway metadata definition. {SMProductName} will ignore Istio Gateways with this annotation, while keeping the automatic management of the other Istio Gateways.
+
+== Disabling automatic route creation for all cases
```

Comment:
Same as above, needs an anchor ID.

Reply:
Made the change, updated the PR, and generated a new preview build.